portal/config/config.py

```diff
@@ -53,10 +53,16 @@ class BaseConfig(object):
 
     # Allow Heroku env vars to override most defaults
     # NB: The value of REDIS_URL may change at any point
-    REDIS_URL = os.environ.get(
-        'REDIS_URL',
-        'redis://localhost:6379/0'
-    )
+
+    # We override REDIS_URL when testing now to avoid needing to
+    # also reset the other variables using it as a default below
+    if os.environ.get('TESTING', 'false').lower() == 'true':
```

Comment:
Would you be able to move this into the [app factory](https://github.com/uwcirg/truenth-portal/blob/develop/portal/factories/app.py#L131), so it can be conditional on `app.testing`, and `os.environ['TESTING']` will only be read once (and set `app.testing`)?

Reply:
I don't believe so.  Can set `app.testing` as you suggest, but we don't yet have an application context when reading the config, so I can't lookup the value of app.testing at this LOC.

Reply:
>Can set app.testing as you suggest
Sorry, should've been more clear. Flask should set `app.testing` if `app.config['TESTING'] == True`. I'm not sure where/when flask does, but we should probably not modify that behavior.

I was hoping we could key all our configuration directly off of `app.testing` instead of some on `app.config['TESTING']`